chromium_src/chrome/common/webui_url_constants.cc

```diff
@@ -10,7 +10,7 @@
 #define kChromeUIAttributionInternalsHost                                     \
   kChromeUIAttributionInternalsHost, kAdblockHost, kAdblockInternalsHost,     \
       kRewardsPageHost, kRewardsInternalsHost, kWelcomeHost, kWalletPageHost, \
-      kTorInternalsHost, kSkusInternalsHost
+      kTorInternalsHost, kSkusInternalsHost, kBraveNewsInternalsHost
```

Comment:
<img width="659" alt="image" src="https://github.com/brave/brave-core/assets/34129/b3b4e9b8-577d-4ce9-8d1c-2c41f4f817ba">
How new is this? I can't access that page on latest nightly

Reply:
looks like it's only available when `kBraveNewsFeedUpdate` is enabled so adding this host to the list should also be be dependent on that feature flag

Reply:
Okay, so just had a quick look - there aren't any other pages in that list which depend on a feature flag, so we have two options:
1. Always show it, and let it error (there are a few other pages which error, but it looks like they're mostly where we turn a Chromium thing off)
2. Don't add this until the flag is gone.

cc @bsclifton

Reply:
Maybe we can just avoid adding until the flag is gone. We can keep it open until then 😄